tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "@ethereumjs/config-typescript",
-  "include": ["src/**/*.ts", "test/**/*.ts"]
+  "include": ["src/**/*.ts", "test/**/*.ts"],
+  "compilerOptions": {
+    "target": "es2020"
```

Comment:
great, we should update `tsconfig.browser.json` to this target as well, which makes me think if we even need a separate browser tsconfig then? I can do a local test and report back.